pages/index.tsx

```diff
@@ -1,22 +1,35 @@
 import type { NextPage } from "next";
-import Link from "next/link";
-import {Container, Image} from "react-bootstrap";
+import { Container, Image } from "react-bootstrap";
 import React from "react";
 
 const Home: NextPage = () => {
   return (
     <Container>
       <Image
-          src="/blueno.png"
-          alt="Blueno logo"
-          width="300"
-          height="300"
-          className="d-inline-block align-top"
+        src="/blueno.png"
+        alt="Blueno logo"
+        width="300"
+        height="300"
+        className="d-inline-block align-top"
       />
       <h1>Wordify</h1>
-        <a href="/game" className="btn btn-secondary btn-lg active" role="button" aria-pressed="true">Play As Guest</a>
-        <br></br>
-        <a href="/#" className="btn btn-secondary btn-lg active" role="button" aria-pressed="true">Log In</a>
+      <a
+        href="/game"
+        className="btn btn-secondary btn-lg active"
+        role="button"
+        aria-pressed="true"
+      >
+        Play As Guest
+      </a>
+      <br></br>
+      <a
+        href="/#"
+        className="btn btn-secondary btn-lg active"
```

Comment:
You can consider using React-Bootstrap's button rather than the basic Bootstrap classnames. i.e., `<Button variant="secondary">Secondary</Button>`